iothub/service/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -99,100 +25,112 @@ public static Task<string> GetExceptionMessageAsync(HttpResponseMessage response
         /// Get the fully-qualified error code from the HTTP response message, if exists.
         /// </summary>
         /// <param name="response">The HTTP response message</param>
-        /// <returns>The fully-qualified error code, or the response status code, if no error code was provided.</returns>
-        public static async Task<ErrorCode> GetExceptionCodeAsync(HttpResponseMessage response)
+        /// <returns>
+        /// The fully-qualified error code, or one derived from the response HTTP status code if no error code was provided.
+        /// </returns>
+        internal static async Task<IotHubErrorCode> GetIotHubErrorCodeAsync(HttpResponseMessage response)
```

Comment:
Do you plan to change the callers to use the other 2 methods directly so we can remove this?